deviceModeInit.js

```diff
@@ -157,23 +152,19 @@ var rudderTracking = (function () {
 
   // TODO: add support for product search
 
+  function sendRudderIdentifierEventPeriodically(cart) {
+    const timeInterval = 50 * 60 * 1000; // 50 mins
+    sendIdentifierToRudderWebhook(cart); // sending rudderIdentifier for the first time
+    setInterval(function () {
+      sendIdentifierToRudderWebhook(cart);
+    }, timeInterval);
```

Comment:
This is not correct. We need to call it periodically. Here we are calling it once.

Reply:
This `setinterval`() calls it after every `timeInterval`

Reply:
Got it